src/main/java/com/esri/samples/geometry/convex_hull_list/ConvexHullListSample.java

```diff
@@ -0,0 +1,255 @@
+      pointCollection1.add(new Point(2409289.65137346, 5477018.71057565));
+      pointCollection1.add(new Point(-2409289.65137346, 5387231.518599));
+      pointCollection1.add(new Point(-2469147.7793579, 8709357.62173508));
+
+      // create a polygon geometry from the point collection
+      Polygon polygon1 = new Polygon(pointCollection1);
+      // create the graphic for polygon1
+      Graphic polygonGraphic1 = new Graphic(polygon1, polygonFill);
+      // add the polygon1 graphic to the graphics overlay collection
+      graphicsOverlay.getGraphics().add(polygonGraphic1);
+
+      // create a point collection that represents polygon2. Use the same spatial reference as the underlying base map.
+      PointCollection pointCollection2 = new PointCollection(SpatialReferences.getWebMercator());
+
+      // add polygon2 boundary map points to the point collection
+      pointCollection2.add(new Point(5993520.19456882, -1063938.49607736));
+      pointCollection2.add(new Point(3085421.63862418, -1383120.04490055));
+      pointCollection2.add(new Point(3794713.96934239, -2979027.78901651));
+      pointCollection2.add(new Point(6880135.60796657, -4078430.90162972));
+      pointCollection2.add(new Point(7092923.30718203, -2837169.32287287));
+      pointCollection2.add(new Point(8617901.81822617, -2092412.37561875));
+      pointCollection2.add(new Point(6986529.4575743, 354646.16535905));
+      pointCollection2.add(new Point(5319692.48038653, 1205796.96222089));
+
+      // create a polygon geometry from the point collection
+      Polygon polygon2 = new Polygon(pointCollection2);
+      // create the graphic for polygon2
+      Graphic polygonGraphic2 = new Graphic(polygon2, polygonFill);
+      // add the polygon2 graphic to the graphics overlay collection
+      graphicsOverlay.getGraphics().add(polygonGraphic2);
+
+      // create a button to create and show the convex hull
+      Button convexHullButton = new Button("Convex Hull");
+      convexHullButton.setMaxWidth(Double.MAX_VALUE);
+
+      // create a button to clear the convex hull results
+      Button resetButton = new Button("Reset");
+      resetButton.setMaxWidth(Double.MAX_VALUE);
+
+      // create a check box for toggling union option on or off
+      CheckBox checkBox = new CheckBox("Union");
+
+      convexHullButton.setOnAction(e -> {
+        // reset the convex hull graphics overlay
+        convexHullGraphicsOverlay.getGraphics().clear();
+
+        Boolean unionBool = checkBox.isSelected();
+
+        // add the geometries of the two polygon graphics to a list of geometries
+        List<Geometry> allPolygonGeometries = new ArrayList<>();
+        allPolygonGeometries.add(polygonGraphic1.getGeometry());
+        allPolygonGeometries.add(polygonGraphic2.getGeometry());
+
+        // retrieve the returned result from the convex hull operation.
+        // when unionBool = true there will be one returned convex hull geometry
```

Comment:
if unioned, one geometry is returned, otherwise one convex hull geometry is returned per input geometry